Mastodon/Scene/HomeTimeline/HomeTimelineViewModel+LoadLatestState.swift

```diff
@@ -92,6 +92,7 @@ extension HomeTimelineViewModel.LoadLatestState {
             }
 
             do {
+                await AuthenticationServiceProvider.shared.fetchAccounts(apiService: viewModel.context.apiService)
```

Comment:
Yeah we have to do something about this. This doesn't feel right. Created [a ticket for this](https://linear.app/mastodon/issue/IOS-216/fix-fetchaccounts-hack).